### PR TITLE
Mark the crate as deprecated

### PR DESCRIPTION
**Pull request by vireshk**
Mark the crate as deprecated in Cargo.toml and update README.md to point to the new crate.

Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>

### CODE REVIEW

**Comment by vireshk** [comment]
Fixes: https://github.com/rust-vmm/vhost-user-backend/pull/88

**Comment by vireshk** [comment]
@andreeaflorescu Hope this is sufficient to mark this deprecated.

File: Cargo.toml
Comment (andreeaflorescu) [inline Cargo.toml:12]: We don't need this because the crate itself is not deprecated, just the repository is.

File: README.md
Comment (andreeaflorescu) [inline README.md:3]: I would say instead that the source code for this crate is moved to: [vhost-user-backend](https://github.com/rust-vmm/vhost/tree/main/crates/vhost-user-backend). You might want to use the root of the repository though in case we change the hierarchy of crates so this can still be reachable.

File: Cargo.toml
Comment (andreeaflorescu) [reply Cargo.toml:12]: Or maybe we should leave it here in case people where consuming the crate from the repository?

File: README.md
Comment (vireshk) [reply README.md]: done

**Review by andreeaflorescu — APPROVED** [review]

**Review by stefano-garzarella — APPROVED** [review]

**Comment by stefano-garzarella** [comment]
Maybe better to have an approval also for the other code owners @jiangliu @sboeuf @slp

**Review by sboeuf — APPROVED** [review]

**Review by slp — APPROVED** [review]

**Comment by andreeaflorescu** [comment]
@stefano-garzarella should we wait for any other approvals or should we merge this?

**Comment by stefano-garzarella** [comment]
> @stefano-garzarella should we wait for any other approvals or should we merge this?

I'd like to have also @jiangliu approval, but since he is also a code owner of [vhost workspace](https://github.com/rust-vmm/vhost/) where this crate is moved, maybe we can merge right now. What do you think?

**Comment by andreeaflorescu** [comment]
> > @stefano-garzarella should we wait for any other approvals or should we merge this?
> 
> I'd like to have also @jiangliu approval, but since he is also a code owner of [vhost workspace](https://github.com/rust-vmm/vhost/) where this crate is moved, maybe we can merge right now. What do you think?

I think we should merge it.

**Comment by stefano-garzarella** [comment]
@andreeaflorescu done! Should we _archive_ this repo?
I tried, but I don't have permissions.

**Pull request merged**
